tests/python/gaia-ui-tests/gaiatest/tests/functional/messages/test_add_to_new_contact_from_messages.py

```diff
@@ -45,7 +45,6 @@ def test_sms_create_new_contact(self):
         new_contact.type_comment(self.contact['note'])
         new_contact.tap_done(return_contacts=False)
 
-        self.wait_for_condition(lambda m: self.message_thread.header_text == self.contact['name'])
```

Comment:
Actually, with the workaround in, it didn't cause failures there anymore, so I just put it back.